tasks/components.py

```diff
@@ -432,6 +432,9 @@ def lint_fxutil_oneshot_test(_):
             if str(file).endswith("_test.go"):
                 continue
 
+            if "cmd/agentless-scanner/main.go" in str(file):
```

Comment:
nit: couldn't this be a plain equality comparison?

Reply:
Done.